app/models/vacancy/publish.rb

```diff
@@ -0,0 +1,10 @@
+class Vacancy::Publish
+  def initialize(vacancy)
+    @vacancy = vacancy
+  end
+
+  def publish
+    @vacancy.save!
+    NotifyVacancyOnSlackWorker.perform_async(@vacancy.id)
```

Comment:
👍 Makes sense.. but I like to test the jobs count.. Is how the doc of sidekiq suggest 😄 
What do you think @lastk ?